modules/tncIdSystem.md

```diff
@@ -1,14 +1,21 @@
-# TNCID UserID Module
+# Overview
 
-### Prebid Configuration
+Module Name: tncIdSystem
+
+# Description
+
+TNCID is a shared persistent identifier that improves user recognition compared to both third-party and first-party cookies. This enhanced identification capability enables publishers and advertisers to consistently recognize their audiences, leading to improved monetization and more precise targeting.  The Newco User ID sub-module adds powerful TNCID user identification technology to your Prebid.js bidstream. 
```

Comment:
please don't commit sales copy to the js repo

Reply:
[Sales description removed](https://github.com/prebid/Prebid.js/pull/12527/commits/b0976ed114f070bc238ed055e049fca7255f4933)